mlrun/db/httpdb.py

```diff
@@ -312,7 +312,8 @@ def connect(self, secrets=None):
                 else server_cfg.get("generate_artifact_target_path_from_artifact_hash")
             )
 
-            config.redis.url = config.redis.url or server_cfg.get("redis.url")
+            config.redis.url = config.redis.url or server_cfg.get("redis_url")
+            config.redis.type = config.redis.type or server_cfg.get("redis_type")
```

Comment:
add comment why you prefer the client to have precedence over the server.

Reply:
added